client/src/lib/chatHelper.js

```diff
@@ -27,15 +29,16 @@ export default (state, app) => {
 		});
 	};
 
-	const handleCopyToClipBoard = async (id) => {
+	const handleCopyToClipBoard = async (id,state, app,key) => {
```

Comment:
Remove the parameter definition, because we already did it here: https://github.com/Dun-sin/Whisper/blob/c6f0687bbb759262e99d23e8ffb40eb0ca9c6afd/client/src/lib/chatHelper.js#L1

The function is inside a function that has those two parameters defined